.github/workflows/test-all-versions.yml

```diff
@@ -26,14 +26,14 @@ jobs:
         ports:
         - 27017:27017
       mssql:
-        image: mcr.microsoft.com/mssql/server:2017-latest
+        image: mcr.microsoft.com/mssql/server:2022-latest
         env:
           SA_PASSWORD: mssql_passw0rd
           ACCEPT_EULA: Y
         ports:
         - 1433:1433
         options: >-
-          --health-cmd "/opt/mssql-tools/bin/sqlcmd -U sa -P $SA_PASSWORD -Q 'select 1' -b -o /dev/null"
+          --health-cmd "/opt/mssql-tools18/bin/sqlcmd -S localhost -U sa -P $SA_PASSWORD -C -Q 'select 1' -b -o /dev/null"
```

Comment:
note for reviewers: that's the path from MSSQL server 2018 and up - as we're using MSSQL server 2022 I assumed there would be a `/opt/mssql-tools22/` directory but inspecting the file system of a running container shows that there's only `/opt/mssql-tools18/`

`-C` ignores the self-signed certificate

Reply:
The best docs I've found for this image is the "Overview" section here: https://hub.docker.com/r/microsoft/mssql-server

> Note Starting with SQL Server 2022 CU 14, we are updating SQL Server 2022 container images to include the new mssql-tools18 package. With the introduction of SQL Server 2022 CU 14, and in all future container images, the previous directory /opt/mssql-tools/bin will be phased out.

Where by "phased out" they mean: just gone. ;)

The version for the "mssql-tools" dir is, at least somewhat, independent of the mssql-server package. I believe that's why it is "mssql-tools18" even for versions of MSSQL server beyond 2018.